tools/site_search.php

```diff
@@ -44,6 +42,17 @@
 echo "<input type='submit' value='" . attr_safe(_("Search")) . "'>";
 echo "</form>";
 
+if ($query && $query != "help") {
+    echo "<p class='warning'>" . sprintf(_("Not sure what to do with '%s', perhaps try one of these."), $query) . "</p>";
```

Comment:
I think we want to `html_safe($query)` here and below

Reply:
Oh yes - good catch! Done and squashed into the relevant (last) commit in the branch.